Tests/src/PublicAPITests/MultiPlayerTests.cpp

```diff
@@ -3040,4 +3040,224 @@ CSP_PUBLIC_TEST(CSPEngine, MultiplayerTests, CreateObjectParentTest)
 	// Log out
 	LogOut(UserSystem);
 }
+#endif
+
+void RunParentDeletionTest(bool Local)
+{
+	SetRandSeed();
+
+	auto& SystemsManager = csp::systems::SystemsManager::Get();
+	auto* UserSystem	 = SystemsManager.GetUserSystem();
+	auto* SpaceSystem	 = SystemsManager.GetSpaceSystem();
+	auto* AssetSystem	 = SystemsManager.GetAssetSystem();
+	auto* Connection	 = SystemsManager.GetMultiplayerConnection();
+	auto* EntitySystem	 = SystemsManager.GetSpaceEntitySystem();
+
+	// Log in
+	csp::common::String UserId;
+	LogIn(UserSystem, UserId);
+
+	// Create space
+	const char* TestSpaceName		 = "CSP-UNITTEST-SPACE-MAG";
+	const char* TestSpaceDescription = "CSP-UNITTEST-SPACEDESC-MAG";
```

Comment:
[Nit] Odd alignment. Could also be `constexpr char*`

Reply:
Alignment looks correct on my end. I'm not sure why its being shown like this

Reply:
I have had issue previously where github is not correctly rendering the file but is fine in checkout, like same thing here

Reply:
It'll be because github is set to render tabs as 8 spaces by default, your editors probably have it at 4, which aligns correctly.

Imo we should have it so tabs are automatically converted to spaces in our linters, but it's not an issue for right now 🤷